WpfPilot.Tests/ElementTests.cs

```diff
@@ -120,5 +122,30 @@ public void TestPublicInterface()
 		AssertImage.ExistsAndCleanup("./checkbox.jpg");
 	}
 
+	[TestCase(0)]
+	[TestCase(1)]
+	public void TestShutdown(int exitCode)
+	{
+		using var appDriver = AppDriver.Launch(ExePath);
+		if (exitCode == 0)
+			Assert.DoesNotThrow(() => appDriver.RunCode(app => app.Shutdown(exitCode)));
```

Comment:
Clean shutdowns no longer throw and are assumed to be a dev gracefully shutting down the app.
Trying to interact with a subsequently shutdown app still throws, as expected.